docs/references/general/grid.md

```diff
@@ -0,0 +1,104 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data produced from FITACF files.
+
+## Naming Conventions
+
+### Single radar GRID files
+
+Currently the common naming convention for a single radar GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+Some radars provide separate data files for each channel. In this case, the channel is specified after the 3-letter station ID:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined "GRD" files for each hemisphere
+
+After GRID files are produced for separate radars, they may then be combined with other radars' GRID files to make a GRD file. This file has the same format as a GRID file, but includes data from multiple radars. 
+See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 
+
+Combined GRD files are produced for the northern and southern hemispheres separately and usually contain 24 hours of data. The common naming convention for 24-hr GRD files is:
```

Comment:
Could this be changed to be a little more vague on the naming convention.  I actually haven't been following this naming convention for the more recent updates to gridding in order to not overwrite data from previous algorithms. 
```suggestion
Combined GRD files are produced for the northern and southern hemispheres separately and usually contain 24 hours of data. One naming convention for 24-hr GRD files could be:
```